core/camel-main/src/test/java/org/apache/camel/main/MainSupportCommandLineTest.java

```diff
@@ -44,20 +45,20 @@ public CamelContext getCamelContext() {
     }
 
     @Test
-    public void testMainSupport() throws Exception {
+    public void testMainSupport() {
         MyMainSupport my = new MyMainSupport();
-        my.run(new String[] { "-d", "1" });
+        Assertions.assertDoesNotThrow(() -> my.run(new String[] { "-d", "1" }));
```

Comment:
Sorry this code is harder to understand and maintain.

Its perfect OK for a test method to have throws exception.
As. the old code is 100% how an end user will write the code.

Reply:
Yes, I didn't mean to touch it. Will rollback it